src/services/kratos/tests-but-not-prod.ts

```diff
@@ -1,22 +1,17 @@
+import { SuccessfulSelfServiceLoginWithoutBrowser } from "@ory/client"
```

Comment:
is this files needed? `src/services/kratos/tests-but-not-prod.ts`

Reply:
yes, used in tests for now. 

more function will be moved in production in follow up PRs.